shm/Cargo.hpp

```diff
@@ -1,12 +1,17 @@
 #pragma once
 
+#include <string>
+
 class Cargo {
 public:
     Cargo& operator+=(const size_t);
     Cargo& operator-=(const size_t);
 
+    bool operator==(const Cargo&) const;
+    bool operator!=(const Cargo&) const;

```

Comment:
We should add the names of function argumets. While reading header you just now what exacly this function does.
```c++
    Cargo& operator+=(const size_t amount);
    Cargo& operator-=(const size_t amount);
```